src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
+import react from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends react.Component {
+  render() {
+    return <Calculator />;
+  }
```

Comment:
Just as [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this milestone, I suggest you make use of functional components instead of class components😉 functional components have fewer complications and are easier to use, unlike class components👍.

<img width="452" alt="image" src="https://user-images.githubusercontent.com/64643995/235630469-50763232-33f9-4f08-9eb4-ab774b1c17cd.png">